include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -442,6 +442,26 @@ MatrixFree<dim, Number, VectorizedArrayType>::internal_reinit(
   // determined in @p extract_local_to_global_indices.
   if (additional_data.initialize_mapping == true)
     {
+      if (dof_handler.size() > 1)
+        {
+          // check if all DoHandlers are in the same hp mode; and if hp
+          // capabilities are enabled: check if active_fe_indices of all
+          // DoFHandler are the same.
+          for (unsigned int i = 0; i < dof_handler.size(); ++i)
```

Comment:
You could start the loop at `i = 1`.

Reply:
You are right. Thanks. I have changed it.